src/registrar/tests/test_reports.py

```diff
@@ -221,8 +223,9 @@ def test_load_full_report(self):
         self.assertEqual(expected_file_content, response.content)
 
 
-class ExportDataTest(TestCase):
+class ExportDataTest(MockEppLib):
```

Comment:
How come you did not need to update what's returned for adomain2 as far as security contact? And if you didn't need to update the test results, why bother with MockEppLib and with cleaning up PublicContact at all?

Reply:
Thanks, good catch